src/content/docs/logs/logs-context/configure-logs-context-go.mdx

```diff
@@ -260,6 +260,63 @@ Otherwise, you can create a context and pass it to logrus:
 txnLogger := log.WithContext(newrelic.NewContext(context.Background(), txn))
 ```
 
+  </Collapser>
+</Collapser>
```

Comment:
@iamemilio this is a mistake I think, two collapser closers in a row but making sure I'm not missing something structurally about that. I can check build later to see also...